util/featureflag/featureflag_test.go

```diff
@@ -54,39 +55,48 @@ func Test_VersionGuard(t *testing.T) {
 	e := gin.Default()
 	e.Use(f1.VersionGuard())
 	e.GET("/ping", func(c *gin.Context) {
-		c.String(200, "pong")
+		c.String(http.StatusOK, "pong")
 	})
 	w := httptest.NewRecorder()
 	req, _ := http.NewRequest("GET", "/ping", nil)
 	e.ServeHTTP(w, req)
 
-	r.Equal(200, w.Code)
+	r.Equal(http.StatusOK, w.Code)
 	r.Equal("pong", w.Body.String())
 
-	// abort with other middlewares
+	// abort
 	handled := false
 	e2 := gin.Default()
 	e2.Use(func(c *gin.Context) {
 		c.Next()
```

Comment:
You can use CreateTestContext to test context directly. No need to use a middleware to test it. Example:

```go
func TestDownload(t *testing.T) {
	handler := New()
	token := handler.mustGetDownloadToken(t, "foobar", "file.txt", time.Second*5)

	r := httptest.NewRecorder()
	c, _ := gin.CreateTestContext(r)
	c.Request, _ = http.NewRequest(http.MethodGet, "/download?token="+token, nil)
	handler.HandleDownloadRequest(c)

	require.Len(t, c.Errors, 0)
	require.Equal(t, http.StatusOK, r.Code)
	require.Equal(t, `attachment; filename="file.txt"`, r.Header().Get("Content-Disposition"))
	require.Equal(t, `application/octet-stream`, r.Header().Get("Content-Type"))
	require.Equal(t, "foobar", r.Body.String())

	// Download again
	r = httptest.NewRecorder()
	c, _ = gin.CreateTestContext(r)
	c.Request, _ = http.NewRequest(http.MethodGet, "/download?token="+token, nil)
	handler.HandleDownloadRequest(c)

	require.Len(t, c.Errors, 1)
	require.True(t, errorx.IsOfType(c.Errors[0].Err, rest.ErrBadRequest))
	require.Contains(t, c.Errors[0].Error(), "Download file not found")
}
```